pkg/server/singleprocess/service_job.go

```diff
@@ -451,16 +508,6 @@ func (s *Service) onDemandRunnerStartJob(
 	}
 	job.Id = id
 
-	// We're going to wait up to 60s for the job be picked up. No reason it won't be
-	// picked up immediately.
-	dur, err := time.ParseDuration("60s")
-	if err != nil {
-		return nil, "", status.Errorf(codes.FailedPrecondition,
-			"Invalid expiry duration: %s", err.Error())
-	}
-
-	job.ExpireTime = timestamppb.New(time.Now().Add(dur))
-
```

Comment:
Note that we no longer expect runners to accept jobs ~60 seconds after a job is queued. This is because pipelines will queue all of the jobs ahead of time at once. In that case, a runner might not get to a job until much later. We instead set up an expire time of 60 seconds once the runner accepts a job.